src/global/mixins.scss

```diff
@@ -55,3 +55,39 @@
     height: 0;
   }
 }
+
+@mixin scrollbar($color: 'light') {
+  /* total width */
+  &::-webkit-scrollbar {
+    width: var(--sbb-spacing-fixed-4x);
+  }
+
+  /* scrollbar itself */
+  &::-webkit-scrollbar-thumb {
+    background-color: var(--sbb-color-black-alpha-30);
+    border: #{px-to-rem-build(7)} solid transparent;
+    border-radius: var(--sbb-border-radius-4x);
+    background-clip: padding-box;
+
+    @if $color == 'dark' {
```

Comment:
It might be easier to use the if function `if($color == 'dark, ... , ...)` (https://www.tutorialspoint.com/sass/if_function.htm)

Reply:
Thanks, though I think it's no longer needed once the `scrollbar-light` and `scrollbar-dark` mixins are introduced.